src/pages/saksoversikt/vilkår/FastOppholdINorge.tsx

```diff
@@ -14,7 +14,7 @@ import { VilkårsvurderingBaseProps } from './types';
 import { Vurdering, Vurderingknapper } from './Vurdering';
 
 interface FormData {
-    status: Nullable<FastOppholdINorgeStatus>;
+    status?: FastOppholdINorgeStatus;
```

Comment:
Jeg foretrekker at man fortsatt har `Nullable` for å være eksplisitt på det, og heller bruke en `?? null` der den initialiseres.
Hvis den må kunne være `undefined` er det bedre å bruke `FastOppholdINorgeStatus | undefined`, sånn at man ikke får lov til å droppe å spesifisere den.

Reply:
✅